### PR TITLE
UI layout improvements

### PR DESCRIPTION
**Pull request by kelilife**

### CODE REVIEW

File: 3ds Max/.cr/personal/FavoritesList/List.xml
Comment (bghgary) [inline 3ds Max/.cr/personal/FavoritesList/List.xml:1]: Is this file supposed to be here?

**Review by bghgary — COMMENTED** [review]
@SergioRZMasson Can you try this out and see if it works for you?

File: 3ds Max/.cr/personal/FavoritesList/List.xml
Comment (SergioRZMasson) [reply 3ds Max/.cr/personal/FavoritesList/List.xml:1]: @bghgary , this seems to work on 3ds Max 2019, UI looks fine.

image

**Review by kelilife — COMMENTED** [review]
Reviewed

File: 3ds Max/.cr/personal/FavoritesList/List.xml
Comment (kelilife) [reply 3ds Max/.cr/personal/FavoritesList/List.xml:1]: The line spacing is a little unequal, and I also had a subtle some places forget to adjust.

**Comment by kelilife** [comment]
UI related content should not be placed in shared projects resulting in unviewable on the designer and not conducive to dedicated maintenance, please use user control libraries instead.

There are still some detail flaws, I'm basing these adjustments on source code changes and have not run tests.

Later, if necessary, I may need to re-initiate a revision to improve the details.

**Comment by bghgary** [comment]
Are you saying this PR is not ready to merge? I'm not sure what you mean. :-)

**Comment by kelilife** [comment]
This request did not finish processing?

**Review by bghgary — APPROVED** [review]

**Pull request merged**